App.js

```diff
@@ -86,6 +87,12 @@ class App extends Component {
                 navigationOptions: {
                     tabBarVisible: false
                 }
+            },
+            [navKeys.MVUM_INSPECTOR]: {
+                screen: MVUMInspectorScreen,
+                navigationOptions: {
+                    tabBarVisible: false
+                }
```

Comment:
GH-104 -- This is the screen that will house the cached MVUM, but for now I can't figure out how to display a local pdf without detaching expo.